lab-guides/01-AzureLocalCloudDeployment/readme.md

```diff
@@ -491,35 +492,40 @@ $LCMCredentials= New-Object System.Management.Automation.PSCredential ($LCMUserN
 
 ### Task05 - Create Azure Resources
 
-Following script will simply create Resource Group and ARC Gateway (optional).
+Following script will simply create Resource Group and Arc Gateway (optional).
 
 ```PowerShell
 $GatewayName="ALClus01-ArcGW"
 $ResourceGroupName="ALClus01-RG"
+
+#List of shortnames for Azure regions (Azure Shell)
+#az account list-locations -o table
```

Comment:
added this pointer for improved clarity on expected values for region / location because some abbreviate west europe as westeu but the ArcInitialization invoke command stricly goes with the region names. In fact westeu is often found.